src/main/java/com/google/sps/servlets/DataFromDatabaseServlet.java

```diff
@@ -58,27 +58,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
-  private void checkTableHasColumns(List<ColumnSchema> columnSchemas) {
```

Comment:
You don't need to change this. Having the method taking in a List (more generic interface than ImmutableList) is the preferred way.

Reply:
Done